src/styles/hackuci.scss

```diff
@@ -30,3 +30,14 @@ h2,
 		box-shadow: 0px 8px 17px rgba(0, 0, 0, 0.37);
 	}
 }
+
+.museum-container {
+	max-width: 720px;
+	@include bootstrap.padding(3rem);
```

Comment:
Question: What's the difference between this and `padding: 3rem;`?

Reply:
The mixin is a shorthand for [Bootstrap's RFS system](https://getbootstrap.com/docs/5.3/getting-started/rfs/) where the amount is calculated based on viewport width.